.github/workflows/test.yml

```diff
@@ -36,6 +36,17 @@ jobs:
         name: pwru
         path: pwru
 
+    - name: Build test-app
+      run: |
+        cd test-app
+        go build .
```

Comment:
I wonder whether we should run `go generate` here to generate `bpf_x86_bpfel.*` rather than adding these generated files to the repo. But then again I'm not sure what the standard practice is regarding these generated files for ebpf-go programs.

Reply:
Yeah, I was considering that, but then from the commit msg:

>  This commit adds the compiled BPF progs, to reduce the runtime dependency burden for the GH workflows (no need to install Clang)

I am expecting that prog to change very rarely, so the no need to install Clang for each PR outweighs the convenience.

Reply:
Should have read the commit message 😬 all good then